utils/fixed_decimal/Cargo.toml

```diff
@@ -46,6 +50,8 @@ path = "src/lib.rs"
 [features]
 default = []
 
+ryu_decimal = ["ryu_floating_decimal"]
```

Comment:
The code looks good now.  I think the only real question is our policy about depending on a hard fork that may or may not have a reliable maintainer.  We may want to reach out personally to dtolnay about getting the functionality into the main crate, or else we may need to adopt the fork, because it's not clear whether the fork has any sort of SLA.

Reply:
(we will discuss this at the next ICU4X meeting)

Reply:
- @zbraniecki suggests sharing our numbers and use case with dtolnay.

@Manishearth to follow up.

Reply:
Followed up; but also it seems like anything I'd want to say there has already been said, so I don't have much hopes on this.

Reply:
I mean, I don't think we can move forward with adding the fork as a dependency of ICU4X, because we have a high bar for dependencies on our fairly low-level library.  I therefore consider this to be an existential issue to our adoption of `ryu`.  I think we need to make a stronger case to share our performance numbers and our use case.

Plan B would be to convert these library functions to examples, showing how one could use `ryu_floating_decimal` in client code.

Reply:
Yeah, but I don't see dtolnay being swayed by that argument :smile: